stac_fastapi/extensions/stac_fastapi/extensions/core/fields.py

```diff
@@ -37,6 +38,10 @@ class FieldsExtension(ApiExtension):
         )
     )
 
+    conformance_classes: List[str] = attr.ib(
```

Comment:
These should be class variables, as they won't change between instances of the class.

Reply:
While that is true in most cases, it is not in all. The filter extension is a good example. As the filter extension is complex, there are varying degrees of implementation. There is a base level to be compliant but then there are other conformance classes that are optional and implementation-specific. Having these as instance variables allows the implementor to specify which classes their implementation conforms to.

https://github.com/radiantearth/stac-api-spec/tree/master/fragments/filter#conformance-classes